lib/ansible/modules/monitoring/zabbix/zabbix_hostgroup_facts.py

```diff
@@ -0,0 +1,159 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# (c) me@mimiko.me
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+RETURN = '''
+---
+hostgroups:
+  description: List of hostgroups.
+  returned: success
+  type: dict
+  sample: [ { "flags": "0", "groupid": "33", "internal": "0", "name": "Hostgruup A" } ]
+'''
+
+DOCUMENTATION = '''
+---
+module: zabbix_hostgroup_facts
+short_description: Gather facts about Zabbix hostgroup
+description:
+   - This module allows you to search for Zabbix hostgroup entries.
+version_added: "2.6"
+author:
+    - "(@redwhitemiko)"
+requirements:
+    - "python >= 2.6"
+    - zabbix-api
+options:
+    server_url:
+        description:
+            - Url of Zabbix server, with protocol (http or https).
+        required: true
+        aliases: [ "url" ]
+    login_user:
+        description:
+            - Zabbix user name, used to authenticate against the server.
+        required: true
+    login_password:
+        description:
+            - Zabbix user password.
+        required: true
+    http_login_user:
+        description:
+            - Basic Auth login
+        required: false
+        default: null
+        version_added: "2.6"
+    http_login_password:
+        description:
+            - Basic Auth password
+        required: false
+        default: null
+        version_added: "2.6"
+    hostgroup_name:
+        description:
+            - Name of the hostgroup in Zabbix.
+            - hostgroup is the unique identifier used and cannot be updated using this module.
+        required: true
+    timeout:
+        description:
+            - The timeout of API request (seconds).
+        default: 10
+'''
+
+EXAMPLES = '''
+- name: Get hostgroup info
+  local_action:
+    module: zabbix_hostgroup_facts
+    server_url: http://monitor.example.com
+    login_user: username
+    login_password: password
+    hostgroup_name:
+      - ExampleHostgroup
+    timeout: 10
+'''
+
+import logging
+import copy
```

Comment:
AFAICS these 2 aren't used anywhere in the code.

Reply:
Thank you @resmo, it's used for our logging internally.
I already removed it now.